src/components/Modal/ModalContainer.tsx

```diff
@@ -1,37 +1,58 @@
 import React, { forwardRef } from 'react'
 
-import { useLocale } from '../../i18n'
 import { ExternalStyles, Theme, useStyles } from '../../styles'
 import { Omit } from '../../util'
-import { Button } from '../Button'
-import { Icon } from '../Icon'
-import { Tooltip } from '../Tooltip'
+import { IconImage } from '../Icon'
+import { IconColor } from '../Icon/Icon'
+import { ModalCloseButton } from './ModalCloseButton'
+import { ModalHeader, SurfaceColor } from './ModalHeader'
 
-export interface ModalContainerProps extends Omit<React.HTMLAttributes<HTMLDivElement>, 'style'> {
+export interface ModalContainerProps extends Omit<React.HTMLAttributes<HTMLDivElement>, 'style' | 'title'> {
   style?: ExternalStyles
   hasCloseIcon?: boolean
   onClose?(): any
+  headerTitle?: string
+  headerSubtitle?: string
+  headerIcon?: IconImage
+  headerIconFill?: IconColor
+  headerIconStroke?: IconColor
+  headerBackgroundColor?: SurfaceColor
+  hasHeaderDivider?: boolean
```

Comment:
```suggestion
  fixedHeader?: boolean
```
Shouldn't the shadow always be visible when the header is fixed? Also, the header should always be fixed?

Reply:
You can choose between a bordered version (created using box-shadow) or a borderless one.
The header can be fixed or scrollable, depending on the implementation by the component’s user.